### PR TITLE
Adding Jack's profile to website

### PR DESCRIPTION
**Pull request by duanxinhuan**
Hi Morford,
Please Check my profile and if you are happy you can add me.

### CODE REVIEW

**Comment by christianmorfordwaitessw** [comment]
Hi @duanxinhuan, 

1. Can you please send your images to a designer to get them formatted correctly. 
Please follow the instructions here [Get and use your official SSW profile photo](https://my.sugarlearning.com/SSW/items/9368/get-and-use-your-official-ssw-profile-photo?term=photos).

File: Jack-Duan/Jack-Duan.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+quote: "These violent delights have violent ends."
+quoteAuthor: "William Shakespeare"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
+
+Jack Duan is a super keen person, he's skill sets are flutter mobile, react and .net framework. In his spare time, he also makes fancy jewelleries
+
+[Design Guidelines](https://www.ssw.com.au/ssw/logo/ssw/)
```
Comment (christianmorfordwaitessw) [inline Jack-Duan/Jack-Duan.md:14]: Remove the design guidelines link

**Comment by duanxinhuan** [comment]
Hi anyone can review this one for me?

File: Jack-Duan/Jack-Duan.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+quote: "These violent delights have violent ends."
+quoteAuthor: "William Shakespeare"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
+
+Jack Duan is a super keen person. His previous work has been split between Mobile, Frontend and Backend.
+
+His skill sets are:
```
Comment (pierssinclairssw) [inline Jack-Duan/Jack-Duan.md:14]: change to either

His skill set includes:

or

His skills include:

File: Jack-Duan/Jack-Duan.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Flutter mobile
+* React
+* Angular
+* .NET framework.
```
Comment (pierssinclairssw) [inline Jack-Duan/Jack-Duan.md:18]: I recommend you change this to .NET

.NET framework is quite old now.

File: Jack-Duan/Jack-Duan.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+In his spare time, he also makes fancy jewellery which he can sell you if you want :)
```
Comment (pierssinclairssw) [inline Jack-Duan/Jack-Duan.md:20]: you need a to after sell

"which he can sell to you if you want"

File: Jack-Duan/Jack-Duan.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+id: "3f8a0335-f08e-ec11-b400-00224818080c"
+
+qualifications: ""
+
+quote: "These violent delights have violent ends."
+quoteAuthor: "William Shakespeare"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
```
Comment (pierssinclairssw) [inline Jack-Duan/Jack-Duan.md:10]: I think you want to get rid of this line

**Review by pierssinclairssw — CHANGES_REQUESTED** [review]

**Pull request closed without merging**